<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>1091</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Ajouter_invites">
     <attribute name="title">
      <string>Ajouter invites</string>
     </attribute>
     <widget class="QLineEdit" name="le_cin">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="la_date">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Num_tel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_date">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_metier">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Metier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Nom">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cin">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_prenom">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pr√©nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_metier">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Num_tel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Num_tel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Mail">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="la_duree">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>320</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Duree">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Duree</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitter_ajout">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Afficher_invites">
     <attribute name="title">
      <string>Afficher invites</string>
     </attribute>
     <widget class="QTableView" name="afficher">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>1041</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Supprimer_invites">
     <attribute name="title">
      <string>Supprimer invites</string>
     </attribute>
     <widget class="QLineEdit" name="le_cin_supp">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cin_supp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitter_supp">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Modifier_invites">
     <attribute name="title">
      <string>Modifier invites</string>
     </attribute>
     <widget class="QLabel" name="label_cin_modif">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Mail_modif">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Duree</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="la_date_modif">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>100</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="la_duree_modif">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_date_modif">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_cin_modif">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="quitter_modif">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>120</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1238</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
